index.html

```diff
@@ -84,9 +84,9 @@ <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
                 <div class="columns small-12 medium-12 large-5 align-self-bottom">
                     <nav class="floe-nav">
                         <ul>
-                            <li><a href="./news/index.html">News</a></li>
-                            <li><a href="./resources.html">Resources</a></li>
-                            <li><a href="./ui-options.html">UI Options</a></li>
+                            <li><a href="./news/index.html">News&nbsp;</a></li>
+                            <li><a href="./resources.html">Resources&nbsp;</a></li>
+                            <li><a href="./ui-options.html">UI Options&nbsp;</a></li>
```

Comment:
You should not be adding `&nbsp;` into the markup. Styling should be done with CSS.